rules-reviewed/jboss-eap/uncategorized/tests/data/MyClass.java

```diff
@@ -12,7 +14,10 @@ public MyClass(){
    public void myMethod() {
 	Class.forName("environment.dependent.MyClass");
         Class.forName("environment.dependent.OtherClass");
-        InitialContext context = new InitialContext();
+        Properties environment = new Properties();
```

Comment:
I think we thank this to Eclipse JDT that does the linking of the method properly :)